src/custom/docs/components/footer/index.tsx

```diff
@@ -1,12 +1,56 @@
 import { QuantinuumLogo } from "../../QuantinuumLogo";
 import { Separator } from "src";
-export const Footer = (props: { subtitle: string, columns: {
```

Comment:
Subtitle here is used for product specific trademark lingo. 
![Screenshot 2024-10-01 at 10 01 10](https://github.com/user-attachments/assets/e4517ee7-6656-4153-81c3-ac9ef00bd6ae)

Reply:
Link added to the **Trademark Guidelines** instead